src/PathCost.py

```diff
@@ -0,0 +1,68 @@
+from chess_enums import Piece
+from chess_enums import Player
+import State
+
+
+class Action:
```

Comment:
This should be class PathCost

Reply:
Late nights make me miss these details, thanks for catching this